src/app/modules/bounce/__tests__/bounce-test-helpers.ts

```diff
@@ -61,9 +62,16 @@ export const makeBounceNotification = (
   recipientList: string[] = [],
   bouncedList: string[] = [],
   bounceType: 'Transient' | 'Permanent' = 'Permanent',
```

Comment:
technically not part of this PR, but could we convert this to an enum too as part of the constants file or types file

Reply:
have added as a TODO!